x-pack/plugins/reporting/server/routes/lib/authorized_user_pre_routing.test.ts

```diff
@@ -145,7 +145,10 @@ describe('authorized_user_pre_routing', function () {
           getMockRequest(),
           getMockResponseFactory()
         )
-      ).toMatchObject({ body: `Sorry, you don't have access to Reporting` });
+      ).toMatchObject({
+        body: `Sorry, you don't have access to Reporting. <a href=mock-link style="font-weight: 600;"
```

Comment:
For a message of this type:

<img width="188" alt="Screen Shot 2022-07-25 at 11 18 28 AM" src="https://user-images.githubusercontent.com/33642766/180847686-446fa01b-0957-49c3-89d3-8ca9fa6ca3e4.png">

We typically use a message like this:

**Permission required**
To create reports, contact your administrator.

Or this:

**Unable to create report**
Contact your administrator for access to reporting features.”

Reply:
There was a discussion about error message improvement [here](https://github.com/elastic/kibana/issues/123058#issuecomment-1108782368). @tsullivan WDYT about current change? See screenshot of in the description.

Reply:
I would go with @gchaps suggestion. What do they see if they click "See the full error?"

Reply:
Should we add link to documentation that explains the details?

Reply:
This is full error view

![DCD9553B-FE11-4DF1-AC32-3F3E039847F2](https://user-images.githubusercontent.com/39378793/181772618-c3730efc-7ac2-4290-9f0b-327d663811af.jpeg)

Reply:
I pulled this branch and found there is a link that explains the details about reporting permissions.
<img width="1665" alt="image" src="https://user-images.githubusercontent.com/908371/181839077-a85140b2-1c96-4f66-928a-7b0e8e954e6b.png">

LGTM!

Reply:
Updated wording. Now it displayed like this.

![6C5CBD8F-2B67-4394-A815-2F418AAD717C_4_5005_c](https://user-images.githubusercontent.com/39378793/182155851-6d942c42-6366-4a23-8c76-7a793b2bf730.jpeg)

Reply:
We typically don't link to the docs when we ask the user to contact their administrator.  

Can the user grant access to reporting themselves or do they need to go through an administrator?  

We should write the message for the preferred action.

Reply:
Changed

![D547F3B2-1B19-49B8-A5F5-0D56E6C8A5B6_4_5005_c](https://user-images.githubusercontent.com/39378793/183052103-395f2137-c7f7-43dc-bf63-9ba1a9f3eb59.jpeg)